nodeEditor/src/diagram/properties/framePropertyComponent.tsx

```diff
@@ -31,7 +39,116 @@ export class FramePropertyTabComponent extends React.Component<IFramePropertyTab
             this.props.frame.onExpandStateChanged.remove(this.onFrameExpandStateChangedObserver);
             this.onFrameExpandStateChangedObserver = null;
         }
-    }    
+    }
+    
+    renderInputBlock(block: InputBlock) {

```

Comment:
I bet this can be shared with what exists in PropertyTabComponent.tsx (Like having a common component to avoid code duplication)